src/components/PhoneNumberInput/index.js

```diff
@@ -1,53 +1,71 @@
 import { h, Component} from 'preact'
-import PhoneNumber, {isValidPhoneNumber} from 'react-phone-number-input'
+import PhoneInput from 'react-phone-number-input/native'
+import {parsePhoneNumberFromString} from 'libphonenumber-js'
 
-import classNames from 'classnames';
-import {localised} from '../../locales'
-
-
-import 'react-phone-number-input/rrui.css'
 import 'react-phone-number-input/style.css'
 import style from './style.css'
 
-const FlagComponent = ({ countryCode, flagsPath }) => (
+import classNames from 'classnames';
+import {localised} from '../../locales'
+
+const FlagComponent = ({ country, flagsPath }) => (
   <span
     className={ classNames('react-phone-number-input__icon', style.flagIcon) }
     style={{
-      'background-image': `url(${ flagsPath }${ countryCode.toLowerCase() }.svg)`,
+      'background-image': `url(${ flagsPath }${ country.toLowerCase() }.svg)`
     }}
   />
 );
 
 class PhoneNumberInput extends Component {
+
   componentDidMount() {
     const { sms, actions } = this.props
-    if (sms && sms.number) {
-      this.validateNumber(sms.number, actions)
+    const initialNumber = sms.number ? sms.number : ""
+    this.validateNumber(initialNumber, actions)
+    this.injectForCountrySelectAriaLabel()
+  }
+
+  injectForCountrySelectAriaLabel = () => {
+    // HACK: This is necessary as react-phone-number-input library is not actually setting country select aria-label
+    const countrySelect = document.getElementsByClassName('react-phone-number-input__country-select')
+    if (countrySelect && countrySelect.length > 0) {
+      countrySelect[0].setAttribute('aria-label', this.props.translate('accessibility.country_select'))
     }
   }
 
   onChange = (number) => {
     const { clearErrors, actions } = this.props
     clearErrors()
-    this.validateNumber(number, actions)
+    const numberString = !number ? "" : number
```

Comment:
Minor: I find it more readable when the condition checks for truthy values first